docs/v1/get-leaderboard-entries.md

```diff
@@ -0,0 +1,63 @@
+<script setup>
+import SampleRequest from '../components/SampleRequest.vue';
+</script>
+
+# Leaderboard Entries
+
+A call to this endpoint will retrieve a given leadboard's entires, targeted by its ID.
+
+[[toc]]
+
+## On-site Representation
+
+A leaderboards's entries can be found on the leaderboard info page:
+
+![Leaderboard Entires](/leaderboard-entries.png)
+
+## HTTP Request
+
+<SampleRequest httpVerb="GET">https://retroachievements.org/API/API_GetLeaderboardEntries.php?i=104370</SampleRequest>
+
+### Query Parameters
+
+| Name | Required? | Description                                                  |
+| :--- | :-------- | :----------------------------------------------------------- |
+| `z`  | Yes       | Your username.                                               |
+| `y`  | Yes       | Your web API key.                                            |
+| `i`  | Yes       | The target leaderboard ID.                                   |
+| `c`  |           | Count, number of records to return (default: 100, max: 500). |
+| `o`  |           | Offset, number of entries to skip (default: 0).              |
+
+## Client Library
+
+Not yet supported.
+
+:::
+
+## Response
+
+::: code-group
+
+```json [HTTP Response]
+{
+  "Count": 100,
+  "Total": 1287,
+  "Results": [
+    {
+      "Rank": 1,
+      "User": "vani11a",
+      "FormattedScore": "390490",
```

Comment:
I'm sorry about this, I know this is going to be really annoying feedback 😞

I think it's great that we have "FormattedScore" in the response. I'm now also thinking we should include "Score" in the response, given directly as the unformatted integer.

Consumers of our endpoints may be used to casting strings like this directly to integers. They may not be aware that it's unsafe here (this could be a speedrun time, or really anything).

If we return both Score and FormattedScore, users can use Score for magic implementation details while still displaying FormattedScore in their UI/UX.